DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/queries/projectingUsingFunctions/projectingUsingFunctions.java

```diff
@@ -0,0 +1,48 @@
+//region Usings
+import net.ravendb.client.documents.session.IDocumentSession;
+import net.ravendb.client.documents.session.IRawDocumentQuery;
+//endregion
+import net.ravendb.demo.common.DocumentStoreHolder;
+
+
+
+import java.util.List;
+
+public class projectingUsingFunctions {
+
+    //region Demo
+    public static class EmployeeDetails {
+        public String FullName;
+        public String Title;
+    }
+    //endregion
+
+
+    public List<EmployeeDetails> run() {
+        List<EmployeeDetails> projectedResults;
+        //region Demo
+        try (IDocumentSession session = DocumentStoreHolder.store.openSession()) {
+        //region Step_1
```

Comment:
indent the `//region Step-x` according to code - see how in other java demos

Reply:
Done